pyright/cli.py

```diff
@@ -6,6 +6,7 @@
 
 from . import node
 from .utils import env_to_bool
+from . import __pyright_version__
```

Comment:
I prefer grouped imports
```suggestion
from . import __pyright_version__, node
from .utils import env_to_bool
```

Reply:
Agreed, the last projects i was working in had isort enabled, so i have stopped thinking about grouping them myself.

Reply:
Yeah I should probably enable isort for this project too.